models/repo_permission_test.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func getUserRepoPermission(repo *repo_model.Repository, user *user_model.User) (Permission, error) {
```

Comment:
Is this function really necessary?

Reply:
agreed this looks like something that a simple search and replace would remove.

Reply:
let the regex began ...

Reply:
... done